bitnami/elasticsearch/templates/master/statefulset.yaml

```diff
@@ -106,7 +106,7 @@ spec:
             - |
               mkdir -p /bitnami/elasticsearch/data
               chown {{ .Values.master.containerSecurityContext.runAsUser }}:{{ .Values.master.podSecurityContext.fsGroup }} /bitnami/elasticsearch/data
-              find /bitnami/elasticsearch/data -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs chown -R {{ .Values.master.containerSecurityContext.runAsUser }}:{{ .Values.master.podSecurityContext.fsGroup }}
+              find /bitnami/elasticsearch/data -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs -r chown -R {{ .Values.master.containerSecurityContext.runAsUser }}:{{ .Values.master.podSecurityContext.fsGroup }}
```

Comment:
Could you explain the `-r` argument what it would do?

Reply:
It helps to avoid an error with empty input, see:

```bash
I have no name!@elastic-elasticsearch-ingest-0:/$ find ./tmp/foo -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs chown -R 1001:1001
chown: missing operand after '1001:1001'
Try 'chown --help' for more information.
I have no name!@elastic-elasticsearch-ingest-0:/$ echo $?
123
I have no name!@elastic-elasticsearch-ingest-0:/$ find ./tmp/foo -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs -r chown -R 1001:1001
I have no name!@elastic-elasticsearch-ingest-0:/$ echo $?
0
```